### ISSUE TITLE
Trying to get pictures to fit on one line

### ISSUE THREAD

**Issue by JiffyBees** [JiffyBees/responsive-section#1]
Hi, I was wondering how I get the 2 images to display correctly in the medium 608px media query it currently has them side by side but without the paragraph and h2. I Know order has to play with it but I am not sure how it works.

@thomasjbradley

**Comment by thomasjbradley** [comment]
I don’t think you have enough elements in your HTML to solve the problem. There should be a `<div>` surrounding each image and another `<div>` surrounding the text. Also, I think the images aren’t in the right order: the two images should come first in the code & then the text last.
